config/manager/kustomization.yaml

```diff
@@ -8,10 +8,10 @@ kind: Kustomization
 images:
 - name: controller
   newName: docker.io/onload/onload-operator
-  newTag: 3.0.0
+  newTag: "3.0"
```

Comment:
The syntax highlighting assumed it was a floating-point number, so I put quotes around it just in case. I'll also try it out without the quotes with Kustomize.

Reply:
Confirmed: Kustomize doesn't like the unquoted version:
```
$ oc kustomize config/default/
error: accumulating resources: accumulation err='accumulating resources from '../manager': '/home/iteterev/lab/kubernetes-onload/config/manager' must resolve to a file': couldn't make target for path '/home/iteterev/lab/kubernetes-onload/config/manager': json: cannot unmarshal number into Go struct field Image.images.newTag of type string
```